pkg/apiserver/resthandler.go

```diff
@@ -679,27 +700,6 @@ func DeleteResource(r rest.GracefulDeleter, checkBody bool, scope RequestScope,
 	}
 }
 
-// queryToObject converts query parameters into a structured internal object by
-// kind. The caller must cast the returned object to the matching internal Kind
-// to use it.
-// TODO: add appropriate structured error responses
-func queryToObject(query url.Values, scope RequestScope, kind string) (runtime.Object, error) {
```

Comment:
The second use of it will be removed in subsequent PR (already under review: #17369 )
This is just cleanup in interfaces.